### PR TITLE
Fixes sensu-plugins/sensu-plugins-supervisor#15

### PR DESCRIPTION
**Pull request by hany**
## Pull Request Checklist

Fixes sensu-plugins/sensu-plugins-supervisor#15

#### General

- [x] Update Changelog following the conventions laid out on [Keep A Changelog](http://keepachangelog.com/)

- [x] Update README with any necessary configuration snippets

- [x] Binstubs are created if needed

- [ ] RuboCop passes

- [ ] Existing tests pass 

#### New Plugins

- [ ] Tests

- [ ] Add the plugin to the README

- [ ] Does it have a complete header as outlined [here](http://sensu-plugins.io/docs/developer_guidelines.html#coding-style)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 ## [1.0.1] 2017-06-01
 ### Fixed
 - check-supervisor-socket.rb: update XMLRPC::XMLParser::XMLStreamParser to XMLRPC::XMLParser::LibXMLStreamParser as per issue https://github.com/sensu-plugins/sensu-plugins-supervisor/issues/16
+- sensu-plugins-supervisor.gemspec: update Gemspec and README as per issue https://github.com/sensu-plugins/sensu-plugins-supervisor/issues/15
```
Comment (majormoses) [inline CHANGELOG.md]: Lets move this above to be be under unreleased.

**Review by majormoses — CHANGES_REQUESTED** [review]
looks good just move the changelog entry to the right place and I will merge and release this.

**Comment by hany** [comment]
@majormoses sorry about that. I misread the changelog. Should be good now as of last release.

**Review by majormoses — APPROVED** [review]
lg2m

**Comment by majormoses** [comment]
released: https://rubygems.org/gems/sensu-plugins-supervisor/versions/1.0.2

**Pull request merged**